README.md

```diff
@@ -54,3 +54,59 @@ Cloudflare Client V4 API Email of the current user.
 to each request.
 
 5. Build an API Client for the Cloudflare Host API which adds the Host Key to all requests.
+
+## config.json 
+
+config.json has the following structure:
```

Comment:
We don't have a `config.json` anymore. It's now `/config` endpont response structure